README.md

```diff
@@ -0,0 +1,10 @@
+ItemShop
+========
+
+Sup playas.
+
+Useful Links
+------------
+- [Ashley Guide](https://github.com/libgdx/ashley/wiki/How-to-use-Ashley)
+- [LibGDX Input](https://github.com/libgdx/libgdx/wiki/Input-handling)
+- [Build Problem](http://stackoverflow.com/questions/3138384/warning-build-path-specifies-execution-environment-j2se-1-4)
```

Comment:
Not sure if you will have the same problem but I used lambdas (see the TestFactory) so I had to specify a certain Java version or something.